metalearners/drlearner.py

```diff
@@ -205,37 +208,85 @@ def evaluate(
         w: Vector,
         is_oos: bool,
         oos_method: OosMethod = OVERALL,
-    ) -> dict[str, float | int]:
-        raise NotImplementedError(
-            "This feature is not yet implemented for the DR-Learner."
+        scoring: Scoring | None = None,
+    ) -> dict[str, float]:
+        safe_scoring = self._scoring(scoring)
+
+        masks = []
+        for tv in range(self.n_variants):
+            masks.append(w == tv)
```

Comment:
```suggestion
```

Reply:
Hahaha, this is my bad, it comes from an old implementation of `evaluate_model` and forgot about them